src/sunmao-helper.ts

```diff
@@ -4,6 +4,52 @@ import { ComponentMetadata } from "@sunmao-ui/core/lib/metadata";
 import { ComponentImplementationProps } from "@sunmao-ui/runtime";
 import { TUnion, TLiteral, Type } from "@sinclair/typebox";
 
+import {
+  ArrayKind,
+  BooleanKind,
+  IntegerKind,
+  NumberKind,
+  ObjectKind,
+  Static,
+  StringKind,
+  UnionKind,
+} from '@sinclair/typebox';
+
+// copy from https://github.com/webzard-io/sunmao-ui/blob/main/packages/runtime/src/utils/parseTypeBox.ts
+// remove generated items by Type.option to get specific example properties
+// mostly used in the case of many attributes like component "Layout.tsx"
+// for some simple properties you should manually specify the specific value for better display
+export function getInitExampleProperties(tSchema: any): Static<typeof tSchema> {
```

Comment:
can import from '@sunmao-ui/runtime'

Reply:
most property in arco are generated by codegen and properties are Type.Optional() will return undefined     
later I will test each component one by one and specify manually exampleProperties, then i will remove this method